### ISSUE TITLE
Mention a method to iterate over all files

### ISSUE THREAD

**Issue by nicorikken** [fsfe/reuse-website#39]
The common use-case for people starting with REUSE is to apply the headers to all files in the repository. The tutorial website doesn't currently hint at such a method, resulting in people wondering how to do so (I've had this question twice after pointing to the official docs). As was mentioned in a REUSE github issue already, this can be done by using `$ git ls-files | xargs reuse addheader ...`. Hinting at the piping options would be helpful to people wanting to adopt REUSE.

I propose to mention it in these locations:
* **Tutorial** in the **[2. Add copyright and licensing information to each file](https://reuse.software/tutorial/#step-2)** section
* [Addheader documentation](https://reuse.readthedocs.io/en/stable/usage.html#addheader) (I know, different repository)

What do you think?

**Comment by mxmehl** [comment]
Definitely, but it would even be better if we had fsfe/reuse-tool#158 done before that.

fsfe/reuse-tool#291 also seems a bit connected to your issue.
